ports/renesas-ra/Makefile

```
@@ -51,6 +51,16 @@ CMSIS_MCU_CAP = RA6M2
 USE_FSP_LPM = 1
 endif
 
+ifeq ($(BOARD),VK_RA6M5)
+BOARD_LOW = vk_ra6m5
+CMSIS_MCU_LOW = ra6m5
+CMSIS_MCU_CAP = RA6M5
+USE_FSP_LPM = 1
+EXPECTED_FSP_HASH = 9713f5b5c3d8bea5f0242feb30c1a57c89ae506a lib/fsp (v2.4.0)
```

Can you please filter this on the series ? So other RA6M5 boards can build ?

```suggestion
ifeq ($(BOARD),RA6M5)
EXPECTED_FSP_HASH = 9713f5b5c3d8bea5f0242feb30c1a57c89ae506a lib/fsp (v2.4.0)
```

---

Yes that makes sense, It will be fixed.